boltrouter/bolt_router.go

```diff
@@ -16,32 +18,78 @@ type BoltRouter struct {
 
 	boltHttpClient     *http.Client
 	standardHttpClient *http.Client
```

Comment:
should we rename this to aws client?

Reply:
You mean the `standardHttpClient`? It is only used to make requests to Quicksilver so I'd say no.

Reply:
ok! was confused why we had a gcp client but not a aws